### ISSUE TITLE
N64 Controller Not Fully Supported

### ISSUE THREAD

**Issue by Roshibomb** [sm64js/sm64js#162]
While the buttons all seem to work just fine, due to the right stick not existing on an N64 controller, the camera and taunt menu are entirely inaccessible (other than bringing up the taunt menu, which is doable, although its functionally useless without a right stick). A proposed fix for this is simply translating all inputs from the right stick into button mappings so it's possible to use the c buttons to control the camera/taunt menu. It should be togglable of course so that people who have controllers with sticks can play the game as well. The c buttons themselves are recognized fine on my controller, but since there is no option to use them in place of a stick, I can't do anything with them.

**Comment by TERRAERROR** [comment]
Did a PR for adding C-Axis to the gamepad
If accepted set X to 5 and Y to 2.

**Comment by Roshibomb** [comment]
There doesn't seem to be a "5" option for C-axis (X _or_ Y), I tried every other option and it didn't seem to work. I might be using a different brand N64 controller than you are, because I only have options 0-3 for both X and Y (and setting Y to 2 doesn't seem to make a difference, either). For reference, my controller has no Nintendo logo (it simply has an oval where it would be) and built-in USB support. It also uses dInput. Finally, according to a d-xInput converter I used, my c buttons are as follows:
C-Left = Button 4
C-Right= Button 9
C-Up = Button 10
C-Down = Button 1
(These are the raw controller inputs that get translated to xInput using the converter, however, since SM64JS recognizes dInput anyway, I don't use the actual converter. I just used it to find button numbers.)
I hope at least one of those things is helpful.
